src/services/session.service.ts

```diff
@@ -113,7 +111,7 @@ export class SessionService {
       return
     }
 
-    return AES.decrypt(encryptedKey, sessionKey).toString(encUtf8)
+    return decryptSeed(encryptedSeed, sessionKey)
```

Comment:
if `sessionKey` is not present, then is there any workflow that is triggered to ask the user for the password?

Reply:
If there is no sessionKey, the extension will return error to the dapp that invoked the function saying the user is not logged in. There is a red exclamation mark on the button of the extension to indicate that the user is not logged in.

I think it's not a good idea to open a popup window on each request when the user is not logged in.